compiler/src/abi/elements.rs

```diff
@@ -243,36 +219,8 @@ mod tests {
         FuncOutput,
         FuncType,
         Function,
-        ModuleABIs,
         VarType,
     };
-    use std::collections::HashMap;
-
-    #[test]
-    fn module_abis_json() {
```

Comment:
I don't think there is any reason to serialize an entire module, so I removed that functionality and the test validating it.